### PR TITLE
Webpack and Node update

### PR DESCRIPTION
**Pull request by richpeters**
Moving forward projects will use node v12.4 and Webpack 4. 

Changes in functionality: 
- Use `core-js@3` for polyfills, uses babel/preset-env `usage` to insert where necessary
-`vue-loader` to enable use of `.vue` files and an appropriate entry file for quick spin up
- a SVG sprite builder
- Remove code formatting (on save)
- Clean `/dist` on save to remove old files
- Only embed data for images / assets smaller than 1kb
- Install nvm using `.nvmrc` file (this is awaiting work on `dev-admin` repo)

### CODE REVIEW

File: {{cookiecutter.project_slug}}/static/src/scss/forms/_forms.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@ form {
     select {
         padding-right: 2.25rem;
         appearance: none !important;
-        background: url(../../img/svg/chevron_down.svg) calc(100% - 0.625rem) 0.75rem / 1.25rem
+        background: url(../../spite/svg/chevron_down.svg) calc(100% - 0.625rem) 0.75rem / 1.25rem
```
Comment (richardsmithdevsoc) [inline {{cookiecutter.project_slug}}/static/src/scss/forms/_forms.scss]: possible typo? `spite` or `sprite`?

File: {{cookiecutter.project_slug}}/webpack.config.js [hunk 2]
```diff
@@ -20,18 +29,30 @@ const config = {
...
     {
         name: 'development',
         entry: config.entry,
         output: config.output,
         plugins: [
+            // Dist clean
+            new CleanWebpackPlugin(),
+
+            // SVG sprite
+            new SVGSpritemapPlugin('./static/src/sprite/*.svg', {
```
Comment (richardsmithdevsoc) [inline {{cookiecutter.project_slug}}/webpack.config.js]: this is cool

File: {{cookiecutter.project_slug}}/fabfile.py [hunk 3]
```diff
@@ -113,6 +113,18 @@ def update():
 
         # Install python/node packages
         run('pip install --quiet --requirement requirements/production.txt')
+
+        # Install nvm using .nvmrc version
+        run('nvm install')
```
Comment (richardsmithdevsoc) [inline {{cookiecutter.project_slug}}/fabfile.py]: does this still need `--no-progess`?

Comment (richpeters) [reply {{cookiecutter.project_slug}}/fabfile.py]: Yeah!

**Pull request merged**